services/lgtm/lgtm.tester.js

```diff
@@ -79,14 +79,14 @@ t.create('alerts: lgtm inaccessible')
 t.create('grade: missing project')
   .get('/grade/java/g/some-org/this-project-doesnt-exist.json')
   .expectJSON({
-    name: 'code quality: java',
-    value: 'project not found',
+    name: 'lgtm',
+    value: 'not found',
   })
 
 t.create('grade: lgtm inaccessible')
   .get('/grade/java/g/apache/cloudstack.json')
   .networkOff()
```

Comment:
Same with tests that use `.networkOff()`. We can remove them because this case is handled by the base classes and has its own tests in core